.circleci/config.yml

```diff
@@ -13,18 +13,23 @@ jobs:
       - run:
           name: External Dependencies
           command: |
-            go get github.com/mattn/goveralls
-            go get -u github.com/jstemmer/go-junit-report
-            curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-            go get -u github.com/haya14busa/goverage
+            GO111MODULE=off go get github.com/mattn/goveralls
+            GO111MODULE=off go get -u github.com/jstemmer/go-junit-report
+            GO111MODULE=off go get -u github.com/haya14busa/goverage
+            GO111MODULE=off go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
The latest version (`v1.18.0`) of golangci-lint is now working with Go 1.13, so we should install it using the [recommended way](https://github.com/golangci/golangci-lint#ci-installation).